biopet-utils/src/main/scala/nl/lumc/sasc/biopet/utils/summary/db/Schema.scala

```diff
@@ -0,0 +1,146 @@
+package nl.lumc.sasc.biopet.utils.summary.db
+
+import java.sql.Date
+
+import slick.driver.H2Driver.api._
+
+/**
+ * Created by pjvan_thof on 27-1-17.
+ */
+object Schema {
+
+  case class Run(id: Int, name: String, outputDir: String, version: String, commitHash: String,
```

Comment:
More documentations would be nice, for example:

* Are there any forbidden characters in `name`? How about empty strings? If the validation is not done here, where is it done?
* What is an example of a `version` string? Do we accept anything or do we expect a pattern?
* Are there any limitations on the commit hash? Is any number of characters fine?

This is for all the classes (tables) defined in this `Schema` object.